.phan/Shopware/Models/Attribute/Order.php

```diff
@@ -5,4 +5,4 @@
 class Order
```

Comment:
Are these still needed?

Reply:
It is in Components/Order/Confirmation.php and bootstrap.php

Reply:
👍